dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *    Copyright 2019 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package io.dockstore.webservice.core;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Entity
+@ApiModel(value = "Image", description = "Image(s) associated with tags and workflow versions")
+@Table(name = "image")
+@SuppressWarnings("checkstyle:magicnumber")
+
+public class Image {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @ApiModelProperty(value = "Implementation specific ID for the image in this webservice", position = 0)
+    private long id;
+
+    @Column(name = "checksums")
+    @ApiModelProperty(value = "Checksum(s) associated with this image", position = 1)
+    private ArrayList<String[]> checksums = new ArrayList<>();
```

Comment:
`List<String[]> is sufficient for the left side
https://www.javaworld.com/article/2073649/why-extends-is-evil.html kinda thing (is old, but still applicable)

Reply:
1. I think this needs to be a `List<Checksum>` or `Set<Checksum>` so that this entity will have the correct model. I know for TRS we could instead do something in ToolsImplCommon.convertEntryToTool, but I still think we would want it to be typed for non-TRS API.
2. @NatalieEO and I looked at this for a little bit, and it looks a little tricky to map this to a Postgres array with Hibernate. Seems like she would need to something like [this](https://vladmihalcea.com/how-to-map-java-and-sql-arrays-with-jpa-and-hibernate/). Just checking if you think we should go down that route.

Reply:
Hmmm, it seems a bit less supported than I would like. 
If the issue is serializing into something typed for the client, then something like https://fasterxml.github.io/jackson-databind/javadoc/2.3.0/com/fasterxml/jackson/databind/annotation/JsonSerialize.html might work. 

If the issue is that we also want something typed coming out of hibernate, I don't know if that's simple.